### PR TITLE
docs: Add Cost estimates section to the README

### LINKED ISSUE #12: Add cost estimation
https://github.com/near/near-lake/pull/25#discussion_r832103661:

> If I recall correctly, @telezhnaya estimated that assuming there are 8 shards (there are only 4 at the moment on mainnet) and block production is stable at 1 block per second, the cost of following each block and chunk in the network is roughly $10/month (paying for the read requests from S3).

It would be great to add real numbers to the README (e.g. we can setup a single node using private credentials and let it run for a week)

### PR DESCRIPTION
**Pull request by khorolets**
Closes #12

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,6 +89,32 @@ Available parameters:
 * `s3_region_name: String` - provide the region for AWS S3 bucket
 * `start_block_height: u64` - block height to start the stream from
 
+## Cost estimates
+
+**TL;DR** approximately $18.15 per month (for AWS S3 access, payed directly to AWS)
```
Comment (telezhnaya) [inline README.md:94]:
```suggestion
**TL;DR** approximately $18.15 per month (for AWS S3 access, paid directly to AWS) for the reading of fresh blocks
```

Comment (telezhnaya) [reply README.md:94]: Major: it's not true since we will make `list` request more frequently :(

File: README.md [hunk 2]
```diff
@@ -89,6 +89,32 @@ Available parameters:
...
+
+And a number of `list` requests we need to perform for a day:
+
+```
+86400 blocks per day / 1000 requests * $0.005 per 1k list requests = $0.432 * 30 days = $12.96
+
+$5.19 + $12.96 = $18.15
+```
+
+The price depends on the number of shards but the difference wouldn't be drastic
```
Comment (telezhnaya) [inline README.md:117]: It linearly depends, so it could be drastic

File: README.md (hunk 1 above)
Comment (khorolets) [reply README.md]: > I think it's a good idea also to provide the cost of reading the data from genesis till some date (we should take into account that 4 shards were introduced just recently)

IMO the most important and most frequent use case is to run an indexer following the tip of the network. So I think if anyone except us needs to read from genesis they can follow the logic of calculations to estimate the cost for their usecase.

**Review by frol — APPROVED** [review]

**Review by telezhnaya — APPROVED** [review]
